### PR TITLE
 Mapping Some Unmapped Classes

### PR DESCRIPTION
**Pull request by MaheEvil**

### CODE REVIEW

**Comment by MaheEvil** [comment]
For reference I will not be looking to fully map out classes, just going there and mapping unnamed classes so that there are sufficient classes named for me to try to more thoroughly map something like world-gen features

**Comment by SpaceWalkerRS** [comment]
> I added `_LOCATION` suffixes to every other ID I mapped but felt like it was unnecessary here

yeah that's fair, imo those names are fine then

File: mappings/1.10.2#16w32a.tinydiff [hunk 1]
```diff
@@ -3665,13 +3665,13 @@ c	net/minecraft/unmapped/C_5684040		net/minecraft/entity/living/mob/hostile/Zomb
...
-	f	Lnet/minecraft/unmapped/C_0561170;	f_9017576		f_9017576
-	f	Lnet/minecraft/unmapped/C_0561170;	f_2555418		f_2555418
-	f	Lnet/minecraft/unmapped/C_0561170;	f_9915429		f_9915429
-	f	Lnet/minecraft/unmapped/C_0561170;	f_6589813		f_6589813
-	f	Lnet/minecraft/unmapped/C_0561170;	f_3627989		f_3627989
+c	net/minecraft/unmapped/C_6977594		net/minecraft/client/render/entity/ZombieVillagerRenderer
+	f	Lnet/minecraft/unmapped/C_0561170;	f_7499748		DEFAULT_LOCATION
+	f	Lnet/minecraft/unmapped/C_0561170;	f_9017576		FARMER_LOCATION
+	f	Lnet/minecraft/unmapped/C_0561170;	f_2555418		LIBRARIAN_LOCATION
+	f	Lnet/minecraft/unmapped/C_0561170;	f_9915429		PREIST_LOCATION
```
Comment (SpaceWalkerRS) [inline mappings/1.10.2#16w32a.tinydiff:3672]: `PREIST` -> `PRIEST`

File: mappings/1.12-pre2#1.12-pre3.tinydiff [hunk 2]
```diff
@@ -959,21 +959,21 @@ c	net/minecraft/unmapped/C_3310297$C_8801857		C_8801857
...
+	f	Ljava/util/List;	f_2523961		recipeTabButtons
+	f	Lnet/minecraft/unmapped/C_3866728;	f_1896807		tabButton
 	f	Lnet/minecraft/unmapped/C_8545792;	f_6718370		f_6718370
 	f	Lnet/minecraft/unmapped/C_8545792;	f_6138568		f_6138568
 	f	Lnet/minecraft/unmapped/C_8545792;	f_9426620		f_9426620
 	f	Lnet/minecraft/unmapped/C_0191817;	f_0136727		f_0136727
-	f	Lnet/minecraft/unmapped/C_6306298;	f_6722717		f_6722717
-	f	Lnet/minecraft/unmapped/C_5217418;	f_3192223		f_3192223
+	f	Lnet/minecraft/unmapped/C_6306298;	f_6722717		inventoryMenu
+	f	Lnet/minecraft/unmapped/C_5217418;	f_3192223		creaftingInventory
```
Comment (SpaceWalkerRS) [inline mappings/1.12-pre2#1.12-pre3.tinydiff:976]: `creafting` -> `crafting`

**Comment by MaheEvil** [comment]
Marked this ready to review because it is new years here and I don't think I am getting much work done soon

**Pull request merged**